Assets/Scripts/helpers.js

```diff
@@ -0,0 +1,11 @@
+﻿(function initializeHelpers(window) {
```

Comment:
Ah yes, finally, we are building our own JavaScript framework! LombiqJS FTW!!

Reply:
lol, hopefully not. The hope is that we can substitute almost all typical jQuery usage with just these 11 lines. That's why the commit says [This is like 90% of what we use jQuery for.](https://github.com/Lombiq/Orchard-Base-Theme/pull/3/commits/8115cb6ea04f80ae23fe67e9db7baccc802ea42f)